css/styles.css

```diff
@@ -93,16 +92,18 @@ button {
   display: flex;
   flex-direction: column;
   flex: 4;
-  background-color: pink;
 }
 
 .artist_view {
-  background-color: white;
-  flex: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  background-image: url(https://s3-us-west-2.amazonaws.com/s.cdpn.io/7022/g-eazy.png);
+  background-size: cover;
+  flex: 0 0 320px;
```

Comment:
This is not supposed to be the final version of the artist view section. It could differ from @amartinezg changes.